neps/nep-0463.md

```diff
@@ -0,0 +1,138 @@
+
+### Benefits:
+
+Standardizing the Sputnik DAO contract would provide:
+
+* greater accessibility
+* better coordination
+* easier development
+
+(compared to not having a standard interface)
+
+#### *What use cases does it support?*
+
+* *complete display of “My DAOs” in wallets and on-chain profiles*
+* *composability across platforms with a common ABI for any `DAO`*
+* *cohesive, yet decentralized governance by NEAR communities*
+
+DAO contract standards help builders create new DAOs that integrate with existing DAOs.
+
+## Rationale
+
+[Sputnik DAOs](https://github.com/near-daos/sputnik-dao-contract) represent a widely used contract system on NEAR, and the code has been thoroughly reviewed and tested in the community. Our hope is to improve accessibility of that robust open-source product by formally introducing a compatible standard. Ultimately, we are limiting complexity of the interface to include only necessary functions.
+
+## Specification 
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The `DAO` contract storage contains `Policy`, `Proposals`, and `Config`.
+
+#### CORE DAO FUNCTIONALITY
+
+* creating and managing:
+  * **proposals**
+  * **policies**
+  * **members**
+* processing **votes** and executing **decisions**
+* updating the **state** of the DAO as a whole
+
+### DAO Interface
+
+```ts
+/// For decentralized autonomous organizations (DAOs).
+///
+/******************/
+/* CHANGE METHODS */
+/******************/
+///
+/// new: create a DAO
+///
+/// Arguments:
+/// * `config`: DAO configuration
+/// * `policy`: members and roles / permissions
+///
+function new(
+  config: string,
+  policy: string,
+);
+///
+/// add_proposal: submit a proposal to the DAO
+///
+/// Requirements
+/// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes.
+/// * Calling the method fails in case of insufficient deposit.
+/// * Only those with permission can add the proposal.
+/// * The proposal is added to a list of proposals.
+///
+/// Arguments:
+/// * `description`: context of the proposal
+/// * `kind`: type of the proposal
+function add_proposal(
+    proposal: {
+        description: string,
+        kind: string,
+    },
+): u64;
+///
+/// act_proposal: act on given proposal by ID, if permissions allow
+///
+/// A successful workflow will end in a success execution outcome
+/// to the callback on the DAO contract at the method `on_proposal_callback`.
+///
+/// Requirements:
+/// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes.
+/// * Only those with permission can act on the the proposal.
+///
+/// Arguments:
+/// * `id`: the ID of the proposal.
+/// * `action`: the action done to the proposal.
+function act_proposal(
+    id: u64,
+    action: string,
```

Comment:
Same here please provide more details and elaborate what this is, I'm sure it's not a string